mne/utils/check.py

```diff
@@ -791,12 +791,13 @@ def _check_qt_version(api='PyQt5'):
             version = qt_mod.__version__
     except Exception:
         version = 'unknown'
-    if sys.platform == 'darwin' and api in ('PyQt5', 'PySide2'):
-        if not _compare_version(version, '>=', '5.10'):
-            warn(f'macOS users should use {api} >= 5.10 for GUIs, got %s. '
-                 'Please upgrade e.g. with:\n\n'
-                 f'    pip install "{api}>=5.10,<5.14"\n'
-                 % (version,))
+    else:
+        if sys.platform == 'darwin' and api in ('PyQt5', 'PySide2'):
+            if not _compare_version(version, '>=', '5.10'):
+                warn(f'macOS users should use {api} >= 5.10 for GUIs, got %s. '
+                     'Please upgrade e.g. with:\n\n'
+                     f'    pip install "{api}>=5.10,<5.14"\n'
+                     % (version,))
```

Comment:
Could you use f-strings here please? I think we do that in most of our new code these days.

Reply:
Do you mean using f-strings for implicit concatenation (e.g. `f'Please upgrade e.g. with:\n\n'` or f-strings to replace the `%` operator? I think the latter should definitely be replaced, but I'm not sure the first is necessary (but it reads nicer).

Reply:
I mean the latter, right now I see an unholy mix of f-strings and `%s`

Reply:
I suggest [d148377](https://github.com/mne-tools/mne-python/pull/10430/commits/d1483778807b9cd15e208e360b6deed14620bb54)

Reply:
Very nice, this avoids the awkward f-string which doesn't contain `{}`!

Reply:
thx!